src/org/zaproxy/zap/extension/sequence/resources/help/contents/sequence.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 3.2 Final//EN">
+<HTML>
+<HEAD>
+<META HTTP-EQUIV="Content-Type" CONTENT="text/html; charset=utf-8">
+<TITLE>
+Sequence Scanner
+</TITLE>
+</HEAD>
+<BODY BGCOLOR="#ffffff">
+
+<H1>Sequence Scanner</H1>
+This add-on facilitates the recording and scanning sequences of requests. 
+In some web applications it is necessary for particular events or pages to be accessed or completed in a certain order.
+By recording a sequence and then scanning it, it is possible to ensure such flows happen in an expected manner.
+
+<H2>Creating Sequences</H2>
+There are two options for creating sequences:
+<br><br>
+<ol>
+<li>In either the Sites Tree or History tab select the requests you wish to have included, right click, and 
+use "Add To Zest Script" (either choosing to create a new script or adding to an existing Sequence script).</li>
+<li>From the main tool bar, use the "Record a New Zest Script..." button, selecting "Sequence" as the type.</li>
+</ol>
+
+<H2>Scanning</H2>
+When a new Active Scan is created there is a "Sequence" tab in which you can choose the sequence(s) to be scanned. 
```

Comment:
Might worth saying that the tab is only shown when the advanced option is enabled.